src/main/scala/com/netflix/edda/mongo/MongoElector.scala

```diff
@@ -85,7 +85,7 @@ class MongoElector extends Elector {
             "ltime" -> null,
             "data" -> Map("instance" -> instance, "id" -> "leader", "type" -> "leader"))))
       // if we got an error then uniqueness failed (someone else beat us to it)
-      isLeader = if (wr.getN() == 0) true else false
+      isLeader = if (wr.getN() == 1) true else false
```

Comment:
I have tested failover with this code but I haven't induced an error condition that would cause this to result in 0, so I'm making some assumptions about it working as documented.